### PR TITLE
Reformatting kotlin codebase/xml layouts

### PR DESCRIPTION
**Pull request by mejdoo**
## What's new in this PR?
Auto formatting the Kotlin code based + new xml layouts
For consistency, I enabled  `kotlinx.android.synthetic.*`imports as they are accepted by detekt
#### APK
[Download build #1868](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1868/artifact/build/artifact/wire-dev-PR2769-1868.apk)
[Download build #1876](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1876/artifact/build/artifact/wire-dev-PR2769-1876.apk)
[Download build #1924](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1924/artifact/build/artifact/wire-dev-PR2769-1924.apk)

### CODE REVIEW

File: app/src/main/kotlin/com/waz/zclient/feature/auth/registration/personal/email/CreatePersonalAccountWithEmailFragment.kt [hunk 1]
```diff
@@ -10,15 +10,13 @@ import com.waz.zclient.core.extension.empty
 import com.waz.zclient.core.extension.replaceFragment
 import com.waz.zclient.core.extension.viewModel
 import com.waz.zclient.feature.auth.registration.di.REGISTRATION_SCOPE_ID
-import kotlinx.android.synthetic.main.fragment_create_personal_account_with_email.confirmationButton
-import kotlinx.android.synthetic.main.fragment_create_personal_account_with_email.createPersonalAccountWithEmailEditText
-import kotlinx.android.synthetic.main.fragment_create_personal_account_with_email.createPersonalAccountWithEmailTextInputLayout
```
[the remark this answers is not in the document]
Comment (mejdoo) [reply app/src/main/kotlin/com/waz/zclient/feature/auth/registration/personal/email/CreatePersonalAccountWithEmailFragment.kt:15]: It's accepted by detekt. I don't see any problem to use wild card imports for layout files.

[the remark this answers is not in the document]
Comment (BradleyWire) [reply app/src/main/kotlin/com/waz/zclient/feature/auth/registration/personal/email/CreatePersonalAccountWithEmailFragment.kt:15]: Then can we update the guide please to acknowledge differences between regular imports and synthetic ones

[the remark this answers is not in the document]
Comment (mejdoo) [reply app/src/main/kotlin/com/waz/zclient/feature/auth/registration/personal/email/CreatePersonalAccountWithEmailFragment.kt:15]: @gizemb we can also link the guide to the jira board for visibilty

[the remark this answers is not in the document]
Comment (BradleyWire) [reply app/src/main/kotlin/com/waz/zclient/feature/auth/registration/personal/email/CreatePersonalAccountWithEmailFragment.kt:15]: Just to add: I review PRs based on the agreed upon style guide regardless of whether we think something is a good or bad idea as decisions that were made previously and we need to respect them. I'm happy with it being changed on this occasion however.

**Comment by mejdoo** [comment]
build again

**Pull request merged**